azerothcore/azerothcore-wotlk
# fix(Core/Script) npc Tom Hegger target player for aggro text

## CHANGES PROPOSED:
-  add if into the case for npc Tom Hegger



## ISSUES ADDRESSED:
-  https://github.com/azerothcore/azerothcore-wotlk/issues/3443 (only addresses the aggro text issue)
- Credits: illfated



## TESTS PERFORMED:
- Test build/run on centos 7
- Check that aggro text shows the players class now instead of $c



## HOW TO TEST THE CHANGES:

- .tele Borean
- .quest add 11794
- use the Oculus of the Exorcist while standing in front of the npc Tom Hegger
- Click on the npc Tom Hegger and select the gossip text "What do you know about the Cult of the Damned?"
- Verify the aggro text now shows your characters class instead of $c. 

## KNOWN ISSUES AND TODO LIST:




## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
